life/updated/02-life.adoc

```diff
@@ -943,169 +943,171 @@ insomuch that I could hardly go on my feet,
-for the Lord was pleased to continue my exercise in that city,
+which is but little of their persecution towards me in consideration of what follows,
+for the Lord was pleased to continue my exercise in that city
 where John Story abode much of his time.
-Several more of that spirit oftentimes frequented there,
+Several more of that dividing spirit oftentimes frequented there,
 and the Lord was pleased to make me so sensible of them,
 that in the night season I had many a sore travail of spirit,
 when I knew not of them by information from any one.
 Then did I cry unto the Lord in secret,
 "`What shall I do to go through such hard things? Oh, that I may be excused,
-or that you will be pleased to keep me in silence this day;
-then should I be very willing to go to meeting to wait upon you,
-and to sit under the shadow of your wing with great delight,
-where your fruit will be pleasant to my taste.`"
-Then would come up before me,
+or that You will be pleased to keep me in silence this day.
+Then should I be very willing to go to meeting to wait upon You,
+and to sit under the shadow of Your wing with great delight,
+where Your fruit will be pleasant to my taste.`"
+Then would come up before me
 the covenant that I made with the Lord in the days of my distress, when all the world,
 and the friendship of it, would not yield one drop of comfort to my poor distressed soul.
 I promised the Lord in that day, twenty years before,
-that if he would redeem my soul from death, and give me assurance of life,
-I would serve him all my days, if he would give me strength, and be with me;
-for I mattered not what I went through for his name`'s sake.
-It would often come up before me, that they who followed the Lord, and loved him most,
-did whatsoever he commanded them.
-I cannot but admire the long forbearance, and lovingkindness of the Lord,
-that he had not cut me off in my gainsaying, and unfaithfulness;
-for I never lacked the assistance of his holy Spirit, in giving up to his requirings,
-blessed be the name of the Lord our God, and the right arm of his strength, forevermore;
-who alone has been our keeper and preserver to this very day;
-glory be to his great name forevermore.
+that if He would redeem my soul from death, and give me assurance of life,
+I would serve Him all my days, if He would give me strength and be with me,
+for it mattered not what I went through for His name`'s sake.
+It would often come up before me that they who followed the Lord, and loved Him most,
+did whatever He commanded them.
+I cannot but admire the long forbearance and lovingkindness of the Lord,
+that He had not cut me off in my gainsaying and unfaithfulness,
+for I never lacked the assistance of His holy Spirit in giving up to His requirings.
+Blessed be the name of the Lord our God and the right arm of His strength forevermore,
+Who alone has been our keeper and preserver to this very day.
```

Comment:
I don't think I ever capitalize who or whom when they refer to God.... at least not where it is clear who it is talking about. Also... on the next line, I kinda like forevermore. I wouldn't kill those. Its old, but it sounds nice, and everybody understands it.